### DISCUSSION TITLE
How to colorize output like in Microsoft Docs

### DISCUSSION THREAD

**Discussion by ialobanov** [PowerShell/PowerShell#18039] [General]
Hi! How I can set up my PowerShell output to view like this:
![image](https://user-images.githubusercontent.com/110660329/188729717-25709530-e234-472f-a23c-55637f32f4c4.png)
Find [here](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/get-eventlog?view=powershell-5.1).
Now I have this:
![image](https://user-images.githubusercontent.com/110660329/188730366-98fda180-4a62-4503-a07d-53070d03e87b.png)

**Comment by Ayanmullick** [comment]
I had a similar question long back. 
https://stackoverflow.com/questions/22617983/retain-coloring-scheme-for-powershell-cmdlet-after-execution

Not sure if this helps.
https://marketplace.visualstudio.com/items?itemName=IBM.output-colorizer

**Reply by ialobanov** [reply]
It can be used for PowerShell? Looks great!

**Reply by Ayanmullick** [reply]
It could colorize the VSC `Output` pane when you are working with repositories etc. However, it doesn't work with the PowerShell output in the Terminal.

![image](https://user-images.githubusercontent.com/17693249/188735416-bb0d1ce3-d984-43ad-a2a3-ede0ee0555d5.png)

**Reply by ialobanov** [reply]
In VSC all good, but I want to change my PowerShell output.

**Reply by Ayanmullick** [reply]
I think one needs to do additional configurations.
https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_ansi_terminals?view=powershell-7.2
